.circleci/config.yml

```diff
@@ -50,18 +37,16 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Install lerna
-          command: npm i --prefix=$HOME/.local -g lerna@6
-      - create_concatenated_package_lock:
-          filename: combined-package-lock.txt
+          name: Install pnpm
+          command: npm i --prefix=$HOME/.local -g pnpm
       - restore_cache:
           keys:
             - deps-v5-{{ .Branch }}-{{ checksum "combined-package-lock.txt" }}
             - deps-v5-{{ .Branch }}
             - deps-v5-
       - run:
           name: Install dependencies
-          command: lerna bootstrap
+          command: pnpm install
```

Comment:
There is an official action that we might wanna use to remove all the manual stuff https://github.com/pnpm/action-setup